setup.py

```diff
@@ -106,6 +111,8 @@
         "pydantic~=1.9",
         "email_validator~=1.2",
         "requests~=2.28",
+        "numpy~=1.23",
+        "h5py~=3.7",
```

Comment:
Should these both still be optional?

Reply:
I am not entirely sure what you mean with this remark.

Did I not edit the setup file correctly so that the NumPy and hdf5 dependencies are not installed by default, or do you want we to change the code so that it won't give an error when these dependencies are not present?

Reply:
This comment picks out the lines where numpy and h5py are listed under `install_requires`, which means they are always installed. You have already added them above as an extra named `hdf5` so these bits are unnecessary.

Reply:
I tried to remove them from "install_requires" but in that case the setup of the docker-image fails. 
So I now removed the hdf5_deps instead.

Reply:
The docker image is failing because you are using `numpy` outside of the hdf5 additions (see https://github.com/Materials-Consortia/optimade-python-tools/pull/1292#discussion_r937663611).

Reply:
I have now added NumPy, to "install_requires", so the setup of the docker image is successful.